lib/prehog/proto/prehog/v1alpha/teleport.proto

```diff
@@ -1,17 +1,3 @@
-// Copyright 2022 Gravitational, Inc
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
```

Comment:
Why remove the license?

Reply:
Ah my bad, when we copy the teleport.proto over from prehog, we need to manually add this and I forgot, re-adding now